.infrastructure/02_iam_group.tf

```diff
@@ -0,0 +1,41 @@
+# External devs read only group
+resource "aws_iam_group" "external_developers_read_only" {
+  name = "external_developers_read_only"
+}
+
+# IAM Group Policy Read Only
+resource "aws_iam_group_policy" "external_developers" {
+  name  = "external_developers_policy"
+  group = aws_iam_group.external_developers_read_only.name
+
+  # Terraform's "jsonencode" function converts a
+  # Terraform expression result to valid JSON syntax.
```

Comment:
Why don't you use the AWS Managed policy
arn:aws:iam::aws:policy/ReadOnlyAccess instead of create your own policy?
Take a look at the terraform resource aws_iam_group_policy_attachment
It should be something like:

`
resource "aws_iam_group_policy_attachment" "test-attach" {
  group      = aws_iam_group.group.name
  policy_arn = "arn:aws:iam::aws:policy/ReadOnlyAccess"
}
`

Reply:
Thanks for the advice! I've changed it 🙏